src/Adapters/Storage/Mongo/MongoStorageAdapter.js

```diff
@@ -171,6 +174,12 @@ export class MongoStorageAdapter implements StorageAdapter {
         });
         this.client = client;
         this.database = database;
+
+        // databaseVersion
+        const adminDb = database.admin();
+        adminDb.serverStatus().then(status => {
+          this.databaseVersion = status.version;
```

Comment:
It would be best to move this to `performInitialization` to ensure the version is available.

Reply:
done